UEBPrep/app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
```

Comment:
@caesarfish Why do we have duplicate functionality between the session controller and the user/homepage controller?